File: Curs2/05.parametri_globali.py


### Parametri locali si globali

x = 10
print(x)

def functie_cu_x_local():
    x = "hello"
    print(x)


functie_cu_x_local()
print(x)


def functie_cu_x_global ():
   global x
   x = "schimbare totala"
   print(x)

functie_cu_x_global()
print(x)

File: Curs2/05.parametri_globali.R


### Parametri locali si globali

x <- 10
print(x)

functie_cu_x_local <- function() {
   x <- "hello"
   print(x)
}

functie_cu_x_local()
print(x)


functie_cu_x_global <- function() {
   x <<- "schimbare totala"
   print(x)

}
functie_cu_x_global()
print(x)
